pkg/component/extensions/network/network_test.go

```diff
@@ -53,6 +53,11 @@ var _ = Describe("#Network", func() {
 		networkPodMask     = 11
 		networkServiceIp   = "100.64.0.0"
 		networkServiceMask = 13
+
+		networkPodV6IP       = "2002:db8:1::"
+		networkPodV6Mask     = 48
+		networkServiceV6IP   = "2002:db8:3::"
+		networkServiceV6Mask = 108
```

Comment:
Please switch to the documentation range ([RFC-3849](https://datatracker.ietf.org/doc/html/rfc3849)):

```suggestion
		networkPodV6IP       = "2001:db8:1::"
		networkPodV6Mask     = 48
		networkServiceV6IP   = "2001:db8:3::"
		networkServiceV6Mask = 108
```